src/styles/card.css

```diff
@@ -0,0 +1,84 @@
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  width: 200px;
+  height: 408px;
+  padding: 32px 16px 16px;
+  background-color: var(--color-white);
+  border: 1px solid var(--color-white-border);
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+}
+
+.card__about {
+  color: var(--color-secondary);
+}
+
+.card__title {
+  margin: 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--color-main-accent);
+}
+
+.card__prod-code {
```

Comment:
you want your styles names maximally clear, so it is better not to use shortcuts